cli/tests/integrations/test_package.py

```diff
@@ -207,7 +207,8 @@ def test_describe():
     "framework"
   ],
   "versions": [
-    "0.8.1"
+    "0.8.1",
+    "0.9.0-RC3"
```

Comment:
It's intended to work this way.  The index JSON in the universe provides a map for each package from unique software version (like 0.8.1) to the largest ordinal package version that provides that software version (like 0).  There's already code to implement the first example flag you mentioned but it needs to be exposed.  It would be useful to be able to specify either the software version or the package version.  For this reason I recommend somehow making it explicit (e.g. --software-version / --package-version) or some better naming scheme.  There was a JIRA for adding this feature at one point but I can't seem to find it right now.

> On Jun 29, 2015, at 21:10, José Armando García Sancio <notifications@github.com> wrote:
> 
> In cli/tests/integrations/test_package.py:
> 
> > @@ -207,7 +207,8 @@ def test_describe():
> >      "framework"
> >    ],
> >    "versions": [
> > -    "0.8.1"
> > +    "0.8.1",
> > +    "0.9.0-RC3"
> I see. Maybe the issues is in how we generate the version in the CLI. Is the plan for the user to maybe do something like dcos package install --version=0.8.1 marathon? or dcos package install --version=0 marathon? @ConnorDoyle?
> 
> —
> Reply to this email directly or view it on GitHub.
>